cloud-stream-sink-null-payload-converter/src/main/resources/application.yml

```diff
@@ -0,0 +1,38 @@
+spring:
+  cloud:
+    function:
+      definition: sink
+    stream:
+      bindings:
+        sink-in-0:
+          destination: TEMPS.Q
+          #The presence of "group" tells the binder to follow the "consumer group" pattern
+          group: SINK
+          consumer:
```

Comment:
consider removing `consumer.concurrency` if not needed. keep things short and simple :)

Reply:
Done